client/reader/stream/style.scss

```diff
@@ -559,3 +567,155 @@
 		margin: 0 0 14px;
 	}
 }
+
+.stream__two-column {
+	align-items: flex-start;
+	display: flex;
+	flex-wrap: nowrap;
+	gap: 80px;
+	justify-content: flex-start;
+
+	.reader-sidebar-site {
+		list-style: none;
+	}
+
+	.reader-sidebar-site_link {
+		align-items: flex-start;
+		display: flex;
+		flex-wrap: nowrap;
+		gap: 12px;
+		justify-content: flex-start;
+		margin-bottom: 24px;
+
+		.count {
+			position: relative;
+			top: 2px;
+		}
+
+		img {
+			/* stylelint-disable-next-line declaration-property-unit-allowed-list */
+			border-radius: 100%;
+			position: relative;
+			top: 4px;
+		}
+	}
+
+	.reader-sidebar-site_sitename {
+		color: var(--color-neutral-100);
+		font-weight: 600;
+		font-size: $font-body-small;
+		line-height: 22px;
+
+		&:hover {
+			color: var(--color-text-subtle);
+		}
+
+		span {
+			display: -webkit-box;
+			overflow: hidden;
+			width: 227px;
+			-webkit-line-clamp: 1;
+			-webkit-box-orient: vertical;
```

Comment:
Can we use properties that are compatible across all browsers we support?

Reply:
A couple of notes here:

1. Despite `-webkit-line-clamp` being prepended with `-webkit` I think it actually has [pretty good coverage across major browsers](https://caniuse.com/?search=-webkit-line-clamp).
2. `-webkit-box-orient` seems to have [good compatibility](https://caniuse.com/?search=-webkit-box-orient) as well.
3. I added `display: block` as a fallback value to `display: -webkit-box` in https://github.com/Automattic/wp-calypso/pull/70667/commits/8ca09f3dc44666d1b600b3ed1fa98cf94c623fe2 just to be safe
4. I tested these update in Firefox after making them to ensure that they work as expected.

Reply:
TIL, and thanks for confirming 👍